controllers/licenseissuer/deploy/Kubefile

```diff
@@ -0,0 +1,16 @@
+FROM scratch
+
+USER 65532:65532
+
+COPY registry registry
+COPY manifests manifests
+
+ENV canConnectToExternalNetwork "true"
+ENV enableMonitor "true"
+ENV CollectorURL "https://anqp0y.laf.dev/CollectorHandle",
+ENV NotificationURL "https://anqp0y.laf.dev/NotificationHandle",
+ENV RegisterURL "https://anqp0y.laf.dev/RegisterHandle",
+ENV CloudSyncURL "https://anqp0y.laf.dev/CloudSyncHandle",
+ENV LicenseMonitorURL "https://anqp0y.laf.dev/LicenseMonitor"
```

Comment:
can you use a regular name for this url?
e.g: `https://licneseissuer.laf.dev`

Reply:
I think it would be more reasonable to rename it to `https://infostream.sealos`

Reply:
This requires applying for a domain name...As of now, it cannot be implemented.

Reply:
map to licenseissuer.sealos.run.